acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/TransactionSignerParamsSupplier.java

```diff
@@ -71,6 +74,10 @@ public TransactionSignerParamsSupplier(
       params.add("multifile-based-signer");
       params.add("--directory");
       params.add(keysDirectory.toAbsolutePath().toString());
+    } else if (metadataLibraryPath != null) {
```

Comment:
metadataDirectory? The cli doesn't refer to this as a library so could be confusing to call it one